docs/data/charts/highlighting/highlighting.md

```diff
@@ -43,15 +43,55 @@ This kind of interaction is controlled by series properties `highlightScope` whi
 
 {{"demo": "ElementHighlights.js"}}
 
-## Controlled highlight
+## Controlled item highlight
 
-The highlight can be controlled by the user when they set `highlightedItem` and `onHighlightChange`.
+The item highlight can be controlled by using `highlightedItem` and `onHighlightChange`.
 
 You can set the `highlightedItem` value based on inputs, and sync it when the user hover over an item themselves.
 
 {{"demo": "ControlledHighlight.js"}}
 
-### Synchronizing highlights
+## Controlled axis highlight
+
+The highlight can be controlled by using `highlightedAxis` prop.
+Its value can be `null` to remove axis highlight, or an object `{ direction: 'x' | 'y', axisId: string, dataIndex: number }`.
+
+The `onAxisInteraction` handler is trigger each time the pointer crosses the boundaries between two axis values.
+Its parameter is an array of objects `{ direction, axisId, dataIndex }`.
+One per axis.
```

Comment:
So, if no axis highlight exists, the argument is an empty array, right?

> One per axis.

How does this work? I'm trying the example below and there's an x and a y axes, but `onAxisInteraction` only returns: 

```json
[
    {
        "axisId": "x-axis",
        "dataIndex": 4
    }
]
```

Which axes are supposed to be shown?